src/mpid/ch4/netmod/ofi/ofi_win.c

```diff
@@ -950,8 +954,18 @@ int MPIDI_OFI_mpi_win_attach_hook(MPIR_Win * win, void *base, MPI_Aint size)
                                         0ULL,   /* In:  flags               */
                                         &mr,    /* Out: memregion object    */
                                         NULL), rc);     /* In:  context             */
-        mpi_errno = MPIDI_OFI_mr_bind(MPIDI_OFI_global.prov_use[0], MPIDI_OFI_WIN(win).mr,
```

Comment:
I think this was a typo from PR #5708. When `MPIDI_OFI_WIN(win).mr` is non-null, it exits on line 929. I believe it should be `mr` in the first place.

Reply:
Ah I see, that makes sense. I was confused as to why the binding here was happening with `MPIDI_OFI_WIN(win).mr`. Thanks for catching this.